pkg/controller/project/project.go

```diff
@@ -159,3 +159,47 @@ func (c *Controller) Run(ctx context.Context, workers int) {
 func (c *Controller) RunningWorkers() int {
 	return c.numberOfRunningWorkers
 }
+
+// This function has been introduced to migrate the legacy project member rolebindings created by older
+// versions of the Gardener dashboard (prior 1.23) to the new .spec.members field of the Project resource.
+// It can be removed in future versions of Gardener.
+func (c *Controller) migrateRoleBindingToProjectMembers() error {
+	projectList, err := c.projectLister.List(labels.Everything())
+	if err != nil {
+		logger.Logger.Errorf("[ROLEBINDING MIGRATION] Error fetching projects: %+v", err)
```

Comment:
Remove logging / use error wrapping to have a pretty message (`go/x/errors` package).